src/index.js

```diff
@@ -178,84 +201,110 @@ class HawkWebpackPlugin {
     }));
   }
 
+  /**
+   * Send git commits
+   *
+   * @param {string} releaseId - release hash
+   * @returns {Promise<Response>}
+   */
+  async sendCommits(releaseId) {
+    const body = new FormData();
+
+    body.append('release', releaseId);
+
+    try {
+      const commits = this.findCommits() || [];
+
+      body.append('commits', JSON.stringify(commits));
+
+      this.log(wrapInColor('Founded some commits', consoleColors.fgGreen), consoleColors.fgCyan, true);
+    } catch (e) {
+      this.log("(⌐■_■) Couldn't get commits: " + e, consoleColors.fgRed);
+
+      return;
+    }
+
+    return this.fetch(body).then(response => {
```

Comment:
so commits will be sent as a separate request? are there any reasons for that?

Reply:
Sours maps are also all sent separately. If something goes wrong with the comments , it will not affect the sourcemaps and vice versa.